workflows/patch-k8s-resource/patch-k8s-resource.sw.yaml

```diff
@@ -11,6 +11,8 @@ functions:
     operation: sysout
   - name: patch
     operation: "specs/patch-k8s-resource.yaml#patchResource"
+    metadata:
+      configKey: ".clusterName"
```

Comment:
so the `configKey` metadata will be used to map the openapi spec URL to the right cluster, it's using the following pattern: `quarkus.rest-client."<normalised_spec_file>.<configKey>".url` am I right?

Is this documented anywhere in sonataflow/openapi generator docs?